ext/NonlinearSolveFixedPointAccelerationExt.jl

```diff
@@ -0,0 +1,56 @@
+module NonlinearSolveFixedPointAccelerationExt
+
+using NonlinearSolve, FixedPointAcceleration, DiffEqBase, SciMLBase
+
+function SciMLBase.__solve(prob::NonlinearProblem, alg::FixedPointAccelerationJL, args...;
+        abstol = nothing, maxiters = 1000, alias_u0::Bool = false,
+        show_trace::Val{PrintReports} = Val(false), termination_condition = nothing,
```

Comment:
```show_trace``` keyword is passed during the method constructing in ```NLsolveJL``` and ```CMINPACK```, I think we'd better keep a unified interface here.

Reply:
I honestly wanted to get rid of those and have them supported similar to https://docs.sciml.ai/NonlinearSolve/stable/basics/solve/#Tracing-Controls. Mostly because, I feel tracing is something that I would occasionally turn on for debugging and having it as a kwarg is more convenient.

But we could not forward those kwargs for non-native solvers

Reply:
Agreed, ```show_trace``` would be more convenient as a kwarg in solve when debugging